components/datatypes/webinfo.schema.json

```diff
@@ -112,6 +112,11 @@
               "meta:titleId": "webinfo##xdm:name##title##72971",
               "meta:descriptionId": "webinfo##xdm:name##description##40881"
             },
+            "xdm:region": {
```

Comment:
What is the purpose of this property ? Does this indicate location in the website where user clicked ? xdm:region is used in address to denote geo region , Please rename this to avoid conflict with xdm:region in address

Reply:
As the description states it is the name of the region where the web interaction occurred. It is the region name in closest DOM proximity to the hyperlink that was clicked. I'm not sure it needs to be more descriptive given that it is encapsulated by webinteraction.

Reply:
sounds good . Can you please rename this property to avoid conflicts and confusion with xdm:region existing in address